.goreleaser.yaml

```diff
@@ -0,0 +1,46 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 2
+
+project_name: festerize
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+archives:
+  - format: zip
+    # this name template makes the OS and Arch compatible with the results of uname.
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    # use zip for windows archives
+    format_overrides:
+      - goos: linux
+        format: tar.gz
+brews:
```

Comment:
Does this mean it's only doing a brew for Macs (or is there a brew for Linux/Windows too)? Or is that `.Os` (above) picking up Macs too?

Reply:
.OS is separate from the brew functionality of gorealeaser. The brew command creates a ruby file and publishes it to the homebrew-festerize repository we have. The os type doesn't impact it

Reply:
I'll ask this at our co-work meeting today, but I'm curious how this works since, I think, running brew on an m1 Mac would need to pull a different binary from running brew on an amd64 Linux machine(?)

Reply:
Nevermind, I see https://github.com/UCLALibrary/homebrew-festerize/blob/main/festerize.rb